src/leaseStatisticReport/components/LeaseStatisticReportForm.js

```diff
@@ -168,15 +167,15 @@ class LeaseStatisticReportForm extends PureComponent<Props, State> {
                   </Column>
                 );
               })}
-              {((reportType !== LeaseInvoicingReportTypes.RENT_FORECAST && reportType !== LeaseInvoicingReportTypes.LEASE_STATISTIC) && fields && !isFetchingOptions) &&<Column small={3} style={{margin: '10px 0'}}>
+              {(!isAsync && fields && !isFetchingOptions) &&<Column small={3} style={{margin: '10px 0'}}>
```

Comment:
Newline before the opening tag of Column would be nice but since there are quite many similar little things here and there, it would take too much time to fix all of those.

Reply:
I agree that there would be a lot to fix in what comes to code formatting...